package.json

```diff
@@ -19,20 +19,23 @@
   "dependencies": {
     "@babel/core": "~7.13.14",
     "@babel/eslint-parser": "~7.13.14",
-    "@lwc/eslint-plugin-lwc": "~1.0.0",
-    "eslint-plugin-import": "~2.22.1",
-    "eslint-plugin-jest": "~23.8.2",
     "eslint-restricted-globals": "~0.2.0"
   },
   "devDependencies": {
+    "@lwc/eslint-plugin-lwc": "~1.0.0",
     "eslint": "^7.22.0",
+    "eslint-plugin-import": "~2.22.1",
+    "eslint-plugin-jest": "~23.8.2",
     "husky": "^4.3.8",
     "lint-staged": "^10.5.4",
     "mocha": "^8.3.2",
     "prettier": "^2.2.1"
   },
   "peerDependencies": {
-    "eslint": "^6 || ^7"
+    "@lwc/eslint-plugin-lwc": "~1.0.0",
```

Comment:
Does it make sense to use `~` for peer dependencies now that they are controlled by consumers? I don't have much experience with this, but my first impression is to use `^`.

Reply:
I think so. It should be tight enough to correspond to the plugin versions that we test against. I originally had this as an exact version but relaxed it a bit.

Reply:
Let's go with this, we can always change this in the future.